sstables/sstable_set.cc

```diff
@@ -363,11 +363,17 @@ class partitioned_sstable_set::incremental_selector : public incremental_selecto
 
 time_series_sstable_set::time_series_sstable_set(schema_ptr schema)
     : _schema(std::move(schema))
-    , _sstables(make_lw_shared<container_t>(position_in_partition::less_compare(*_schema))) {}
+    , _reversed_schema(_schema->make_reversed())
```

Comment:
I refrain from using `get_reversed()` in this series to avoid surprise crashes due to tests without `cql_test_env` executing code paths using `get_reversed()`. I plan on lifting the dependence of the schema registry on `cql_test_env`, then switch to `get_reversed()` everywhere.

Reply:
Ok